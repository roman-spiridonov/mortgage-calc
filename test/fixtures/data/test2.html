<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="author" content="Roman Spiridonov">
  <title>Test</title>
  <script type="text/javascript" async
          src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_CHTML">
  </script>
</head>
<body>
\begin{equation}
\begin{aligned}
\tilde Ba_{\overline{m}|G_m} &amp;= \tilde B\left[(1+G_m)^{-1}+(1+G_m)^{-2}+\cdots+(1+G_m)^{-m}\right] \\
&amp;= \tilde B(1+G_m)^{-1}\left[\frac{(1+G_m)^{-m}-1}{(1+G_m)^{-1}-1}\right] \\
&amp;= \frac{\tilde B}{G_m}\left[1 - (1+G_m)^{-m}\right]
\end{aligned}
\end{equation}
</body>
</html>