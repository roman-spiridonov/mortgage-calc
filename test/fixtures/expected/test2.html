<html lang="en"><head>
  <meta charset="UTF-8">
  <meta name="author" content="Roman Spiridonov">
  <title>Test</title>
  <script type="text/javascript" async="" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-MML-AM_CHTML">
  </script>
<style type="text/css">
                            .mjpage .MJX-monospace {
                            font-family: monospace
                            }

                            .mjpage .MJX-sans-serif {
                            font-family: sans-serif
                            }

                            .mjpage {
                            display: inline;
                            font-style: normal;
                            font-weight: normal;
                            line-height: normal;
                            font-size: 100%;
                            font-size-adjust: none;
                            text-indent: 0;
                            text-align: left;
                            text-transform: none;
                            letter-spacing: normal;
                            word-spacing: normal;
                            word-wrap: normal;
                            white-space: nowrap;
                            float: none;
                            direction: ltr;
                            max-width: none;
                            max-height: none;
                            min-width: 0;
                            min-height: 0;
                            border: 0;
                            padding: 0;
                            margin: 0
                            }

                            .mjpage * {
                            transition: none;
                            -webkit-transition: none;
                            -moz-transition: none;
                            -ms-transition: none;
                            -o-transition: none
                            }

                            .mjx-svg-href {
                            fill: blue;
                            stroke: blue
                            }

                            .MathJax_SVG_LineBox {
                            display: table!important
                            }

                            .MathJax_SVG_LineBox span {
                            display: table-cell!important;
                            width: 10000em!important;
                            min-width: 0;
                            max-width: none;
                            padding: 0;
                            border: 0;
                            margin: 0
                            }

                            .mjpage__block {
                            text-align: center;
                            margin: 1em 0em;
                            position: relative;
                            display: block!important;
                            text-indent: 0;
                            max-width: none;
                            max-height: none;
                            min-width: 0;
                            min-height: 0;
                            width: 100%
                            }</style></head>
<body>
<span class="mjpage mjpage__block"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="100ex" height="16.509ex" style="vertical-align: -7.671ex; max-width: 600;" viewBox="0 -3805.2 43055.4 7108.1" role="img" focusable="false" xmlns="http://www.w3.org/2000/svg" aria-labelledby="MathJax-SVG-1-Title">
<title id="MathJax-SVG-1-Title">\begin{equation}
\begin{aligned}
\tilde Ba_{\overline{m}|G_m} &amp;= \tilde B\left[(1+G_m)^{-1}+(1+G_m)^{-2}+\cdots+(1+G_m)^{-m}\right] \\
&amp;= \tilde B(1+G_m)^{-1}\left[\frac{(1+G_m)^{-m}-1}{(1+G_m)^{-1}-1}\right] \\
&amp;= \frac{\tilde B}{G_m}\left[1 - (1+G_m)^{-m}\right]
\end{aligned}
\end{equation}</title>
<defs aria-hidden="true">
<path stroke-width="1" id="E1-MJMAIN-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path>
<path stroke-width="1" id="E1-MJMAIN-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path>
<path stroke-width="1" id="E1-MJMAIN-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path>
<path stroke-width="1" id="E1-MJMATHI-42" d="M231 637Q204 637 199 638T194 649Q194 676 205 682Q206 683 335 683Q594 683 608 681Q671 671 713 636T756 544Q756 480 698 429T565 360L555 357Q619 348 660 311T702 219Q702 146 630 78T453 1Q446 0 242 0Q42 0 39 2Q35 5 35 10Q35 17 37 24Q42 43 47 45Q51 46 62 46H68Q95 46 128 49Q142 52 147 61Q150 65 219 339T288 628Q288 635 231 637ZM649 544Q649 574 634 600T585 634Q578 636 493 637Q473 637 451 637T416 636H403Q388 635 384 626Q382 622 352 506Q352 503 351 500L320 374H401Q482 374 494 376Q554 386 601 434T649 544ZM595 229Q595 273 572 302T512 336Q506 337 429 337Q311 337 310 336Q310 334 293 263T258 122L240 52Q240 48 252 48T333 46Q422 46 429 47Q491 54 543 105T595 229Z"></path>
<path stroke-width="1" id="E1-MJMAIN-7E" d="M179 251Q164 251 151 245T131 234T111 215L97 227L83 238Q83 239 95 253T121 283T142 304Q165 318 187 318T253 300T320 282Q335 282 348 288T368 299T388 318L402 306L416 295Q375 236 344 222Q330 215 313 215Q292 215 248 233T179 251Z"></path>
<path stroke-width="1" id="E1-MJMATHI-61" d="M33 157Q33 258 109 349T280 441Q331 441 370 392Q386 422 416 422Q429 422 439 414T449 394Q449 381 412 234T374 68Q374 43 381 35T402 26Q411 27 422 35Q443 55 463 131Q469 151 473 152Q475 153 483 153H487Q506 153 506 144Q506 138 501 117T481 63T449 13Q436 0 417 -8Q409 -10 393 -10Q359 -10 336 5T306 36L300 51Q299 52 296 50Q294 48 292 46Q233 -10 172 -10Q117 -10 75 30T33 157ZM351 328Q351 334 346 350T323 385T277 405Q242 405 210 374T160 293Q131 214 119 129Q119 126 119 118T118 106Q118 61 136 44T179 26Q217 26 254 59T298 110Q300 114 325 217T351 328Z"></path>
<path stroke-width="1" id="E1-MJMATHI-6D" d="M21 287Q22 293 24 303T36 341T56 388T88 425T132 442T175 435T205 417T221 395T229 376L231 369Q231 367 232 367L243 378Q303 442 384 442Q401 442 415 440T441 433T460 423T475 411T485 398T493 385T497 373T500 364T502 357L510 367Q573 442 659 442Q713 442 746 415T780 336Q780 285 742 178T704 50Q705 36 709 31T724 26Q752 26 776 56T815 138Q818 149 821 151T837 153Q857 153 857 145Q857 144 853 130Q845 101 831 73T785 17T716 -10Q669 -10 648 17T627 73Q627 92 663 193T700 345Q700 404 656 404H651Q565 404 506 303L499 291L466 157Q433 26 428 16Q415 -11 385 -11Q372 -11 364 -4T353 8T350 18Q350 29 384 161L420 307Q423 322 423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 181Q151 335 151 342Q154 357 154 369Q154 405 129 405Q107 405 92 377T69 316T57 280Q55 278 41 278H27Q21 284 21 287Z"></path>
<path stroke-width="1" id="E1-MJMAIN-AF" d="M69 544V590H430V544H69Z"></path>
<path stroke-width="1" id="E1-MJMAIN-7C" d="M139 -249H137Q125 -249 119 -235V251L120 737Q130 750 139 750Q152 750 159 735V-235Q151 -249 141 -249H139Z"></path>
<path stroke-width="1" id="E1-MJMATHI-47" d="M50 252Q50 367 117 473T286 641T490 704Q580 704 633 653Q642 643 648 636T656 626L657 623Q660 623 684 649Q691 655 699 663T715 679T725 690L740 705H746Q760 705 760 698Q760 694 728 561Q692 422 692 421Q690 416 687 415T669 413H653Q647 419 647 422Q647 423 648 429T650 449T651 481Q651 552 619 605T510 659Q492 659 471 656T418 643T357 615T294 567T236 496T189 394T158 260Q156 242 156 221Q156 173 170 136T206 79T256 45T308 28T353 24Q407 24 452 47T514 106Q517 114 529 161T541 214Q541 222 528 224T468 227H431Q425 233 425 235T427 254Q431 267 437 273H454Q494 271 594 271Q634 271 659 271T695 272T707 272Q721 272 721 263Q721 261 719 249Q714 230 709 228Q706 227 694 227Q674 227 653 224Q646 221 643 215T629 164Q620 131 614 108Q589 6 586 3Q584 1 581 1Q571 1 553 21T530 52Q530 53 528 52T522 47Q448 -22 322 -22Q201 -22 126 55T50 252Z"></path>
<path stroke-width="1" id="E1-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path>
<path stroke-width="1" id="E1-MJMAIN-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
<path stroke-width="1" id="E1-MJMAIN-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path>
<path stroke-width="1" id="E1-MJMAIN-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path>
<path stroke-width="1" id="E1-MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path>
<path stroke-width="1" id="E1-MJMAIN-22EF" d="M78 250Q78 274 95 292T138 310Q162 310 180 294T199 251Q199 226 182 208T139 190T96 207T78 250ZM525 250Q525 274 542 292T585 310Q609 310 627 294T646 251Q646 226 629 208T586 190T543 207T525 250ZM972 250Q972 274 989 292T1032 310Q1056 310 1074 294T1093 251Q1093 226 1076 208T1033 190T990 207T972 250Z"></path>
<path stroke-width="1" id="E1-MJMAIN-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
<path stroke-width="1" id="E1-MJSZ1-5B" d="M202 -349V850H394V810H242V-309H394V-349H202Z"></path>
<path stroke-width="1" id="E1-MJSZ1-5D" d="M22 810V850H214V-349H22V-309H174V810H22Z"></path>
<path stroke-width="1" id="E1-MJSZ3-5B" d="M247 -949V1450H516V1388H309V-887H516V-949H247Z"></path>
<path stroke-width="1" id="E1-MJSZ3-5D" d="M11 1388V1450H280V-949H11V-887H218V1388H11Z"></path>
</defs>
<g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)" aria-hidden="true">
<g transform="translate(41775,0)">
<g id="mjx-eqn-1">
 <use xlink:href="#E1-MJMAIN-28"></use>
 <use xlink:href="#E1-MJMAIN-31" x="389" y="0"></use>
 <use xlink:href="#E1-MJMAIN-29" x="890" y="0"></use>
</g>
</g>
<g transform="translate(8225,0)">
<g transform="translate(-11,0)">
<g transform="translate(167,0)">
<g transform="translate(-11,0)">
<g transform="translate(0,2811)">
 <use xlink:href="#E1-MJMATHI-42" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-7E" x="212" y="577"></use>
<g transform="translate(759,0)">
 <use xlink:href="#E1-MJMATHI-61" x="0" y="0"></use>
<g transform="translate(529,-210)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="28" y="0"></use>
<g transform="translate(0,143)">
 <use transform="scale(0.574)" xlink:href="#E1-MJMAIN-AF" x="-70" y="0"></use>
<g transform="translate(167.44338267193748,0) scale(0.9974586530209003,1)">
 <use transform="scale(0.574)" xlink:href="#E1-MJMAIN-AF"></use>
</g>
 <use transform="scale(0.574)" xlink:href="#E1-MJMAIN-AF" x="651" y="0"></use>
</g>
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-7C" x="935" y="0"></use>
<g transform="translate(858,0)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.574)" xlink:href="#E1-MJMATHI-6D" x="968" y="-185"></use>
</g>
</g>
</g>
</g>
</g>
<g transform="translate(3368,0)">
<g transform="translate(0,2811)">
 <use xlink:href="#E1-MJMAIN-3D" x="277" y="0"></use>
<g transform="translate(1334,0)">
 <use xlink:href="#E1-MJMATHI-42" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-7E" x="212" y="577"></use>
</g>
<g transform="translate(2260,0)">
 <use xlink:href="#E1-MJSZ1-5B"></use>
<g transform="translate(417,0)">
 <use xlink:href="#E1-MJMAIN-28" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="389" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="1112" y="0"></use>
<g transform="translate(2112,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(3620,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,412)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-31" x="778" y="0"></use>
</g>
</g>
 <use xlink:href="#E1-MJMAIN-2B" x="5236" y="0"></use>
 <use xlink:href="#E1-MJMAIN-28" x="6237" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="6626" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="7349" y="0"></use>
<g transform="translate(8350,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(9858,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,412)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-32" x="778" y="0"></use>
</g>
</g>
 <use xlink:href="#E1-MJMAIN-2B" x="11474" y="0"></use>
 <use xlink:href="#E1-MJMAIN-22EF" x="12474" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="13869" y="0"></use>
 <use xlink:href="#E1-MJMAIN-28" x="14870" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="15259" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="15982" y="0"></use>
<g transform="translate(16983,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(18491,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,412)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="778" y="0"></use>
</g>
</g>
</g>
 <use xlink:href="#E1-MJSZ1-5D" x="20569" y="-1"></use>
</g>
</g>
<g transform="translate(0,579)">
 <use xlink:href="#E1-MJMAIN-3D" x="277" y="0"></use>
<g transform="translate(1334,0)">
 <use xlink:href="#E1-MJMATHI-42" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-7E" x="212" y="577"></use>
</g>
 <use xlink:href="#E1-MJMAIN-28" x="2093" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="2483" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="3205" y="0"></use>
<g transform="translate(4206,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(5714,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,412)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-31" x="778" y="0"></use>
</g>
</g>
<g transform="translate(7274,0)">
 <use xlink:href="#E1-MJSZ3-5B"></use>
<g transform="translate(528,0)">
<g transform="translate(120,0)">
<rect stroke="none" width="7125" height="60" x="0" y="220"></rect>
<g transform="translate(60,770)">
 <use xlink:href="#E1-MJMAIN-28" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="389" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="1112" y="0"></use>
<g transform="translate(2112,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(3620,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,362)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="778" y="0"></use>
</g>
</g>
 <use xlink:href="#E1-MJMAIN-2212" x="5504" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="6504" y="0"></use>
</g>
<g transform="translate(193,-780)">
 <use xlink:href="#E1-MJMAIN-28" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="389" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="1112" y="0"></use>
<g transform="translate(2112,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(3620,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,288)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-31" x="778" y="0"></use>
</g>
</g>
 <use xlink:href="#E1-MJMAIN-2212" x="5236" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="6237" y="0"></use>
</g>
</g>
</g>
 <use xlink:href="#E1-MJSZ3-5D" x="7893" y="-1"></use>
</g>
</g>
<g transform="translate(0,-2325)">
 <use xlink:href="#E1-MJMAIN-3D" x="277" y="0"></use>
<g transform="translate(1056,0)">
<g transform="translate(397,0)">
<rect stroke="none" width="1627" height="60" x="0" y="220"></rect>
<g transform="translate(434,676)">
 <use xlink:href="#E1-MJMATHI-42" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-7E" x="212" y="577"></use>
</g>
<g transform="translate(60,-726)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
</g>
</g>
<g transform="translate(3201,0)">
 <use xlink:href="#E1-MJSZ1-5B"></use>
<g transform="translate(417,0)">
 <use xlink:href="#E1-MJMAIN-31" x="0" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2212" x="722" y="0"></use>
 <use xlink:href="#E1-MJMAIN-28" x="1723" y="0"></use>
 <use xlink:href="#E1-MJMAIN-31" x="2112" y="0"></use>
 <use xlink:href="#E1-MJMAIN-2B" x="2835" y="0"></use>
<g transform="translate(3836,0)">
 <use xlink:href="#E1-MJMATHI-47" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="1112" y="-213"></use>
</g>
<g transform="translate(5344,0)">
 <use xlink:href="#E1-MJMAIN-29" x="0" y="0"></use>
<g transform="translate(389,412)">
 <use transform="scale(0.707)" xlink:href="#E1-MJMAIN-2212" x="0" y="0"></use>
 <use transform="scale(0.707)" xlink:href="#E1-MJMATHI-6D" x="778" y="0"></use>
</g>
</g>
</g>
 <use xlink:href="#E1-MJSZ1-5D" x="7422" y="-1"></use>
</g>
</g>
</g>
</g>
</g>
</g>
</g>
</svg></span>

</body></html>